product_profile/models/product_profile.py

```diff
@@ -67,11 +66,11 @@ class ProductProfile(models.Model):
         help="An explanation on the selected profile\n"
         "(not synchronized with product.template fields)",
     )
-    type = fields.Selection(
+    detailed_type = fields.Selection(
```

Comment:
You should provide migration script for this

Reply:
@rousseldenis  , Correct, I forgot it.
I am not very experienced with those scripts. 
(I've noticed the module migration g uide does not provide info on this either).
So please check if this is ok now.